arch-wiki-docs.py

```diff
@@ -33,5 +34,16 @@
 
     downloader.download_images()
 
+    for f in os.listdir(args.output_directory):
+        lang_dir = join(args.output_directory, f)
+        if os.path.isdir(lang_dir):
+            main_page = join(lang_dir, 'Main_page.html')
+            index_html = join(lang_dir, 'index.html')
```

Comment:
These should still be `os.path.join` right? Also please use double quotes for strings.